### DISCUSSION TITLE
How do i make the top bar widgets float on top of the wallpaper/background?

### DISCUSSION THREAD

**Discussion by JuanDavid006** [end-4/dots-hyprland#668] [Q&A]
I love your dotfiles and it's visually stunning except for one thing... the borders on the topbar. I saw another discussion regarding disabling fake corners. But how do i remove the top bar background and just make the widgets float on the screen? Thanks :D

**Comment by end-4** [comment]
so transparent bar background? change [this line](https://github.com/end-4/dots-hyprland/blob/1e297a54d1aaeaf16c97ffe8bc0e1f9702927d78/.config/ags/scss/_bar.scss#L16) to 
```scss
    background-color: rgba(0,0,0,0);
```

**Comment by JuanDavid006** [comment]
It works! But it makes the whole bar transparent. Which is what i want. But I'd also like to make the widgets have their own background like in this picture. How can i do this? Thanks end-4!

![swappy-20240703_131644](https://github.com/end-4/dots-hyprland/assets/123331132/929ed65d-6f95-4088-8314-ffcb7c09064d)


This is what I'm getting:

![swappy-20240703_133232](https://github.com/end-4/dots-hyprland/assets/123331132/427d2b11-2dcb-4245-a57a-f14784b177ca)
